tests/testthat/_snaps/bump_version/NEWS2.md

```diff
@@ -12,4 +12,5 @@
 
 # tea 0.0.0.9000
 
-* Added a `NEWS.md` file to track changes to the package.
+- Added a `NEWS.md` file to track changes to the package.
```

Comment:
Is this an artifact of the pandoc roundtrip?

Reply:
yes

Reply:
let me do an experiment though

Reply:
now 100% sure that it is :-)